### ISSUE TITLE
Add ability to import master realm

### ISSUE THREAD

**Issue by kkovarik** [keycloak/keycloak#12544]
### Description

Thanks for the ability to import realm during startup (https://github.com/keycloak/keycloak/issues/10216) like 
`start --import-realm`

its a step in right direction, as in our deployment, we need fully automated installation, like we had up & running with "old" wildfly based (like 16.x).

What I am struggling now, is unability to import master realm this way. Seems like it gets always created before the import itself?
`
quarkus-keycloak-1               | 2022-06-16 08:59:44,732 INFO  [org.keycloak.services] (main) KC-SERVICES0003: Not importing realm master from file /opt/keycloak/bin/../data/import/master-realm.json.  It already exists.
`

Is there something I am missing? Can I configure keycloak not to create master realm?

### Motivation

To have automatic setup, same as we had with "old" wildfly based keycloak.

**Comment by pedroigor** [comment]
@kkovarik The server does not allow overriding realms when importing at startup but only when running the `import` command. The same behavior if you use "-Dkeycloak.import" when running the legacy distribution.

This is not the first time people ask about this behavior and I think we can include it.

I'm not sure though if we should be opinionated and just override existing realms or if we should provide an option to set whether or not realms should be kept/removed if they exist.

Another point against introducing this now is the upcoming changes to the new store.

**Comment by kkovarik** [comment]
@pedroigor After some digging in source code I found out the old properties do still apply. So was able to achieve the old behaviour using the same properties as in legacy
`
-Dkeycloak.migration.action=import -Dkeycloak.migration.provider=dir -Dkeycloak.migration.dir=/opt/keycloak/realm-config -Dkeycloak.migration.strategy=IGNORE_EXISTING
`

Is this gonna be kept in the future? If yes maybe it can be just documented better. That not so much has changed in the keycloak itself :) 

Reason why I haven't used **kc.sh import** is that I was fighting with db connection for some reason (like the env variables weren't picked up), when using **kc.sh start** these system properties work like a charm. :)

**Comment by pedroigor** [comment]
Yeah, it should work and that is pretty much what the `import` command does but without forcing you to set system properties.

We would like people to not rely on those system properties anymore but on the available commands or the `import-realm` option.

I would say you can keep doing that way but considering the upcoming changes to the new store. I don't think we are going to remove support for the approach you are using anytime soon.

Thanks.

**Comment by pedroigor** [comment]
@ic2hrmk If you start the server like that it should not shutdown after import:

```
kc.sh -Dkeycloak.migration.action=import -Dkeycloak.migration.provider=dir -Dkeycloak.migration.dir=/tmp/export -Dkeycloak.migration.strategy=IGNORE_EXISTING start --http-enabled true --hostname-strict false
```

We should start looking what we can do here to improve UX because, as I mentioned before, using those system properties are not really the best. At the same time, we want to provide a good UX.

I'll start a discussion about this.

**Comment by ic2hrmk** [comment]
Hello @pedroigor, thank you for the example! These new import settings are the puzzle for now, so it's good to have the access to previous system properties (even if they are not the prettiest in the UX) 😃
